package.json

```diff
@@ -25,13 +25,19 @@
     "path-to-regexp": "^1.0.0"
   },
   "devDependencies": {
-    "istanbul": "^0.4.2",
-    "mocha": "^2.0.0"
+    "codecov": "^3.6.5",
+    "eslint": "^6.8.0",
+    "eslint-config-standard": "^14.1.1",
```

Comment:
Not that I have a real problem with the approach, but is there a reason not to just use `standard` on its own?

Reply:
I can't speak for here, but I typically use eslint directly because it plays really nice with all the editors that just understand eslint.

Reply:
FWIW, the same can be true with `standard` in my experience (vim and vscode).  I think the main goal is to reduce maintenance overhead, and having this many deps to keep up-to-date as well as the config files is much more work than just a single dep.

Reply:
Interesting, I will need to check my editor. Less deps is definitely better.

Reply:
yeah just IDE integration. i wish standard just made these dev deps honestly

Reply:
Which IDE are you using?  The vs code plugin for standard support is really good in my experience.  If it is not working for you I am sure we can engage Feross and the standard maintainers to see if there is a better way to support these editors.

Reply:
I use VS Code, but last I tried the Standard JS plugin it pops open the console with an error every time I open a non-StandardJS project, which is quite irritating. I double-checked it's still an issue with the latest version and there is at least one open issue on GitHub regarding it. The eslint plugin is nice and quiet when loading something not using StandardJS, but works when you do, making it appear very seamless.

Reply:
Hm, I am not seeing that behavior, but also I think I did change some settings and have been actively moving things to standard because maintaining eslint configs is the *last* thing I want to be doing 🤣.